dc-cudami-server/dc-cudami-server-backend-jdbi/src/main/java/de/digitalcollections/cudami/server/backend/impl/jdbi/identifiable/entity/HeadwordEntryRepositoryImpl.java

```diff
@@ -41,8 +41,8 @@ public class HeadwordEntryRepositoryImpl extends EntityRepositoryImpl<HeadwordEn
   public static final String TABLE_NAME = "headwordentries";
 
   @Override
-  protected String getSqlSelectAllFieldsJoins() {
-    return super.getSqlSelectAllFieldsJoins()
+  protected String getSqlSelectReducedFieldsJoins() {
+    return super.getSqlSelectReducedFieldsJoins()
```

Comment:
Warum ist dieser Join bei den reduced fields bereits nötig? Selektiert wird aus dem HeadwordRepo doch nur bei all fields (`getSqlSelectAllFields`).

Reply:
weil das headword mit label gleich dabei sein soll (ist essentiell bei HeadwordEntry, auch in Listen)

Reply:
Aber das Headword wird doch im `getSqlSelectReducedFields` gar nicht selektiert. Folglich ist es nur bei retrieveOne dabei und der Join sollte nur im `getSqlSelectFullFieldsJoins` stehen.

Reply:
ok, siehe additional fields joins in identifiablerepo